docs/internationalization.md

```
@@ -0,0 +1,74 @@
+make i18n-download-and-compile
+```
+
+
+## Contributing as a translator or proof-reader
+
+We use the [Crowdin](https://crowdin.com) web platform to translate Messages to different languages.
+It allows translators and proof-readers to contribute on translations in the languages they master.
+
+### Sign-up on Crowdin
+
+If you don't have an account on Crowdin already, go to https://accounts.crowdin.com/register and
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)